project/lib/image_library.py

```diff
@@ -1,8 +1,60 @@
-"""
-A library for all functions image-related
-"""
-
-
-# Turn an image file into an array of features
-def get_image_features(image_path):
-    pass
+"""
+A library for all functions image-related
+"""
+import cv2
+import numpy as np
+from scipy.misc import imread
+
+
+
+# Turn an image file into an array of features, returns 0s if fails
+def get_image_features(image_path):
+    
+    features = extract_features(image_path)
+    resize_array(features)
+    if(check_size(features) == 1):
+        return features
+    else:
+        return [np.zeros(64) for i in range(200)]
+    #pass
+
+#ensures array is size 200*64
+def resize_array(array):
+    num_vectors = len(array)
+    for i in range(num_vectors):
+        #ensure all vectors are length 64
+        v_size = array[i].size
+        if(v_size < 64):
+            array[i] = np.concatenate([array[i], np.zeros(64 - v_size)])
+    if(num_vectors < 200):
+        zero_vector = [np.zeros(64) for i in range(200-num_vectors)]
+        array = np.concatenate([array, zero_vector])
+    return array
+
+#returns 1 if array is correct size, else 0
+def check_size(array):

```

Comment:
I think it would be better to make this return a boolean instead of 1 or 0. Well written tho!